cluster/mesos/docker/util.sh

```diff
@@ -61,7 +61,7 @@ function cluster::mesos::docker::docker_compose {
 # to `docker pull` which always hits the remote docker repo, even if the image
 # is already cached.
 function cluster::mesos::docker::docker_compose_lazy_pull {
-  for img in $(grep '^\s*image:\s' "${provider_root}/docker-compose.yml" | sed 's/[ \t]*image:[ \t]*//'); do
+  for img in $(grep '^\s*image:\s' "${provider_root}/docker-compose.yml" | sed 's/[ \t]*image:[ \t]*//' | grep -vw "mesosphere-local"); do
```

Comment:
We need this or have to push the image to docker hub. It feels strange to me to push test images there, already now for kubernetes-mesos.